test/snoowrap.spec.js

```diff
@@ -209,6 +210,45 @@ describe('snoowrap', function () {
     });
   });
 
+  describe('.fromApplicationOnlyAuth', () => {
+    it('throws a TypeError if no userAgent is provided in node', function () {
+      if (isBrowser) {
+        return this.skip();
+      }
+      expect(() => {
+        snoowrap.fromApplicationOnlyAuth({clientId: 'bar', deviceId: 'baz'});
+      }).to.throw(TypeError);
+    });
+    it('throws a TypeError if no clientId is provided', () => {
+      expect(() => {
+        snoowrap.fromApplicationOnlyAuth({userAgent: 'bar'});
+      }).to.throw(TypeError);
+    });
+    it('throws a TypeError if no grantType is provided', () => {
+      expect(() => {
+        snoowrap.fromApplicationOnlyAuth({userAgent: 'bar'});
+      }).to.throw(TypeError);
+    });
+    (isBrowser ? it.skip : it)('returns a Promise for a valid installed requester', async () => {
+      const newRequester = await snoowrap.fromApplicationOnlyAuth({
+        userAgent: oauthInfo.user_agent,
+        grantType: snoowrap.grantType.INSTALLED_CLIENT,
+        clientId: oauthInfo.installed_app_client_id,
+        deviceId: oauthInfo.device_id
+      });
+      expect(await newRequester.getHot('redditdev', {limit: 1})).to.have.lengthOf(1);
+    });
+    (isBrowser ? it.skip : it)('returns a Promise for a valid client requester', async () => {
+      const newRequester = await snoowrap.fromApplicationOnlyAuth({
+        userAgent: oauthInfo.user_agent,
+        grantType: snoowrap.grantType.CLIENT_CREDENTIALS,
+        clientId: oauthInfo.client_id,
+        clientSecret: oauthInfo.client_secret
+      });
+      expect(await newRequester.getHot('redditdev', {limit: 1})).to.have.lengthOf(1);
```

Comment:
I noticed that these tests are skipped in a browser. What's the reasoning for that? (I don't think any browser tests run automatically, but it's possible to run `npm run test:browser` to open a browser and run the tests.)

Reply:
I was mostly just using `fromAuthCode` tests as a template. Is there a reason that last test is  disabled in browser?

Reply:
I don't remember 😕 

I suppose we could change it to `it.only` and try it out in a browser to see if there are any problems.